### PR TITLE
Updated the Drag effect in the Draggable list

### PR DESCRIPTION
**Pull request by natashapl**
### This pull request contains the following changes:

- The drag effect of the draggable items were updated so that there's a shadow overlay and a drop zone. 

[PR Preview](https://federalist-16f2aca2-467c-449f-b725-5f1a0bd22dcd.sites.pages.cloud.gov/preview/gsa-tts/atj-platform/fix-draggable-list-animation-effect/)

### CODE REVIEW

File: packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx [hunk 1]
```diff
@@ -58,47 +66,109 @@ export const DraggableList: React.FC<DraggableListProps> = ({
...
           })}
         </SortableContext>
+
+        <DragOverlay>
+          {activeId ? (
+            <SortableItemOverlay>
+              {arrayChildren[order.indexOf(activeId)]}
+            </SortableItemOverlay>
+          ) : null}
+        </DragOverlay>
```
Comment (danielnaab) [inline packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx:95]: Nice approach!

File: packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx [hunk 2]
```diff
@@ -58,47 +66,109 @@ export const DraggableList: React.FC<DraggableListProps> = ({
...
   const { attributes, listeners, setNodeRef, transform, transition } =
     useSortable({ id });
-  const context = useFormManagerStore(state => state.context);
+  const context = useFormManagerStore((state) => state.context);
 
   return (
     <div
-      className={`${styles.draggableListWrapper} draggable-list-item-wrapper bg-white margin-bottom-3`}
+      className={`${styles.draggableListWrapper} draggable-list-item-wrapper bg-white margin-bottom-3 ${
+        isActive ? 'dragging' : ''
```
Comment (danielnaab) [inline packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx:156]: Where is the `dragging` class defined?

**Review by danielnaab — APPROVED** [review]
This is awesome... especially like how you were able to keep it all encapsulated in `DraggableList`.

File: packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx (hunk 2 above)
Comment (natashapl) [reply packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx]: I originally added that class name along with "over" with the intention of putting the styles in the module css. Unfortunately, the styles weren't being picked up, so I ended up adding them inline. I removed the class names.

**Pull request merged**